finagle-mysql/src/main/scala/com/twitter/finagle/mysql/Request.scala

```diff
@@ -258,6 +231,20 @@ case class ExecuteRequest(
     }
 }
 
+object ExecuteRequest {
+  def apply(stmtId: Int, params: IndexedSeq[Parameter] = IndexedSeq.empty, hasNewParams: Boolean = true, flags: Byte = 0) = {
```

Comment:
please keep line length < 100.

if you need to do this sanitization for every request, you should disallow creation otherwise.  does this need to be here though?  could we just put it in the constructor of ExecuteRequest?